### DISCUSSION TITLE
signer

### DISCUSSION THREAD

**Discussion by sorenammd** [wevm/viem#1447] [General]
i am trying to use smart accounts (acount abstraction by ERC4337), and for that I need to use signer. it seem that viem uses account now instead of signer. wouldy you please let me know how can I use signer in this case. See attached screenshot for more information. 
![image](https://github.com/wagmi-dev/viem/assets/107199800/a06a43e1-d18d-4675-94a3-37d75aa93de6)

**Comment by ventada** [comment]
the best way it's to not use veim and keep using ehters  :)

**Reply by jxom** [reply]
Let's try keep the discussion constructive. :)

**Comment by jxom** [comment]
I don't believe the Biconomy SDK supports Viem yet. Best to open a discussion on their repository to introduce support for Viem. :)

**Reply by sorenammd** [reply]
Hello. it is not only for Biconomy. The use of ERC 4337 will be increased surely, and this feature (i.e. using signer) will be useful in veriety of users and applications. Please let us know if you are planning to develop this feature.
